skyportal/facility_apis/sedmv2.py

```diff
@@ -0,0 +1,233 @@
+from . import FollowUpAPI
+from baselayer.app.env import load_env
+from datetime import datetime, timedelta
+import json
+import requests
+
+from ..utils import http
+
+env, cfg = load_env()
+
+
+def validate_request_to_sedmv2(request):
+    """Validate FollowupRequest contents for SEDMv2 queue.
+
+    Parameters
+    ----------
+    request: skyportal.models.FollowupRequest
+        The request to send to SEDM.
+
+    method_value: 'new', 'edit', 'delete'
+        The desired SEDMv2 queue action.
+    """
+
+    for param in [
+        "observation_choices",
+        "exposure_time",
+        "exposure_counts",
+        "maximum_airmass",
+        "minimum_lunar_distance",
+        "priority",
+        "start_date",
+        "end_date",
+    ]:
+        if param not in request.payload:
+            raise ValueError(f'Parameter {param} required.')
```

Comment:
Probably would benefit from some bounds checking here. I suppose it's the job of the receiver of this to deal bounds but if we can ensure that we are only sending over valid requests then it's worth doing the checking here too.

Reply:
Added!